transcriptic/commands.py

```diff
@@ -703,7 +705,7 @@ def compile(protocol_name, args):
     call(["bash", "-c", command + " " + ' '.join(args)])
 
 
-def launch(api, protocol, project, save_input, local, accept_quote, params, pm=None, test=None, pkg=None):
+def launch(api, protocol, project, save_input, local, accept_quote, params, test_inputs, pm=None, test=None, pkg=None):
```

Comment:
fyi this is causing a pylint failure in travis:
```
875,12: Too many positional arguments for function call (too-many-function-args)
```
You could turn it off by adding #pylint: disable=too-many-function-args

This is suggesting we're having too many function arguments here, and should break this function up. Something to consider, but probably not high priority for this diff

Reply:
I did notice this and was a bit confuse why it was erroring. Sounds good.